src/ts/localization/languages/en.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Placeholder comment for previous discussion about format. https://github.com/bitmovin/bitmovin-player-ui/pull/271#discussion_r333871498